src/content/guides/advanced-cluster-autoscaler-configuration/index.md

```diff
@@ -0,0 +1,84 @@
+
+You can customize two of these configuration options on a per cluster basis through a ConfigMap inside your clusters. The ConfigMap is named `cluster-autoscaler-user-values` and is located in the `kube-system` namespace.
+
+__Note:__ This feature is only available in more recent cluster versions. To check if your cluster version supports customization through the ConfigMap, you can check if the above-mentioned ConfigMap is present.
+
+```nohighlight
+$ kubectl -n kube-system get cm cluster-autoscaler-user-values
+NAME                                   DATA      AGE
+cluster-autoscaler-user-values                    0         11m
+```
+
+On cluster creation the ConfigMap is empty and below-mentioned defaults will be applied to the final Cluster Autoscaler deployment. To customize any of the configuration options, you just need to add the respective line(s) in the data field of the user ConfigMap.
+
+__Warning:__ Please do not edit any of the other Cluster Autoscaler related resources. Only the user ConfigMap is safe to edit.
+
+## Scale down utilization threshold
+
+The `scale-down-utilization-threshold` defines the proportion between requested resources and capacity, which under the value Cluster Autoscaler will trigger the scaling down action.
+
+```yaml
+data:
+  scale-down-utilization-threshold: 0.65
+```
+
+Our default value is 65%, which means in order to scale down, one of the nodes has to have less utilization (CPU/memory) than this threshold.
+
+# Expander
+
+When there are several node groups managed by the Cluster Autoscaler we can influence which one will be selected to scale up the cluster when it is needed. Our default is `least-waste`. 
+
+- `random`: Select a random node group to scale up.
+- `most-pods`: Select the node group which is the best fit for all pods.
+- `least-waste`: Select the node group which leaves the least fraction of CPU and Memory).
+- `price`: Select the node group which is the most cost-effective.
+- `priority`: Select the node group based on an user-configured priorities assigned to group names.
+
+```yaml
+data:
+  expander: "random"
+```
+
+# Scan Interval
+
+Define what interval is taken to review the state for taking a decision to scale up/down. Our default value is 10 seconds.
+
+```yaml
+data:
+  scanInterval: "100s"
+```
+
+# Skip system pods
+
+By default the Cluster Autoscaler will never delete nodes which run pods of kube-system namespace (but daemonset or mirror ones). It can be modified by setting the property to `false`.
+
+```yaml
+data:
+  skipNodesWithSystemPods: "false"
+```
+
+# Skip pods local storage
+
+The Cluster Autoscaler configuration by default deletes nodes with pods using local storage (`hostPath` or `emptyDir`)". In case you want to disable this action, you need to set the property to `true`.
```

Comment:
```suggestion
The Cluster Autoscaler configuration by default deletes nodes with pods using local storage (`hostPath` or `emptyDir`). In case you want to disable this action, you need to set the property to `true`.
```